documentation/asciidoc/topics/proc_exchanging_sa_tokens.adoc

```diff
@@ -9,16 +9,42 @@ At **NYC** you add the service account token for **LON**.
 
 .Prerequisites
 
-* Get tokens from each service account.
+* Create a service account token secret file as follows.
 +
-Use the following command or get the token from the {osweb}:
+.sa-token.yaml
+[source,yaml,options="nowrap",subs=attributes+]
+----
+apiVersion: v1
+kind: Secret
+metadata:
+  name: ispn-xsite-sa-token #<1>
+  annotations:
+    kubernetes.io/service-account.name: "lon" #<2> 
+type: kubernetes.io/service-account-token 
+----
+<1> Specifies the secret name.
+<2> Specifies an existing service account name.
+
+* Create the secret in your {openshiftshort} cluster.
 +
 [source,options="nowrap",subs=attributes+]
 ----
-oc sa get-token lon
+oc -n <namespace> create -f sa-token.yaml
+----
++
+WARNING: The token created by this procedure never expires.
 
-eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9...
+* Get the service account token.
++
+[source,options="nowrap",subs=attributes+]
 ----
+oc -n <namespace> get secrets ispn-xsite-sa-token -o jsonpath="{.data.token}" | base64 -d
+
+eyJhbGciOiJSUzI1NiIsImtpZCI6IiJ9
+----
+
+TIP: For bounded service account token, use the command `oc -n <namespace> create token <service account>`.
```

Comment:
would it make sense to create its own adoc file and sub-section for this type of token?